src/main-process/appWindow.ts

```diff
@@ -176,6 +178,7 @@ export class AppWindow {
     const { devUrlOverride } = this;
     if (devUrlOverride) {
       const url = `${devUrlOverride}/${urlPath}`;
+      this.rendererReady = true; // TODO: Look into why dev server ready event is not being sent to main process.
```

Comment:
Is this change related to the PR description? Or added accidentally by the merge?